contracts/ics/base/Cargo.toml

```diff
@@ -8,9 +8,6 @@ repository = "https://github.com/synapseweb3/ibc-ckb-contracts"
 
 [dependencies]
 ckb-std = "0.13.0"
-ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "58f256c" }
-axon-tools = { git = "https://github.com/axonweb3/axon.git", package = "axon-tools", rev = "06340ba4", version = "0.1.1", features = ["proof"] }
-axon-types = { git = "https://github.com/axonweb3/axon-contract", rev = "8c2338a" }
+ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "963bf1a4ca5538" }
```

Comment:
```suggestion
ckb-ics-axon = { git = "https://github.com/synapseweb3/ckb-ics.git", rev = "4e89e76d16" }
```

Reply:
The changes between these commits should not affect the contracts.